src/config.c

```diff
@@ -2099,46 +2099,58 @@ static int numericBoundaryCheck(typeData data, long long ll, const char **err) {
     return 1;
 }
 
+#define CONFIGSETCOMMON(data, ll, prev, err) \
```

Comment:
this macro needs to mention "numeric"
```suggestion
#define NUMERIC_CONFIGSET_COMMON(data, ll, prev, err) \
```

I don't see anything in that macro that would function wrong if given the wrong type (i.e. signed or unsigned).
I think it can be integrated back into the numericConfigSet function.
i.e. it'll first do either `memtoull` or `string2ll` and then all these checks on the result (which will always be a plain `long long`).

Reply:
@oranagra I have updated the code for this. Please take a look.